### PR TITLE
InItial dashboard config for rhobs instance utlization dashboard

### PR DESCRIPTION
**Pull request by coleenquadros**
PR to initiate RHOBS instance utlization dashboard. This PR only contains the creation, title and description of the dashboard. Panels  for components will be added as a part of other tickets workflow

Ticket - https://issues.redhat.com/browse/RHOBS-619
Dashboard - https://grafana.stage.devshift.net/d/8-QwLUwVz/rhobs-instance-utilization?orgId=1

### CODE REVIEW

File: observability/dashboards/rhobs-instance-utilization.libsonnet [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+              hide: 0,
+              includeAll: false,
+              label: 'datasource',
+              multi: false,
+              name: 'datasource',
+              options: [],
+              query: 'prometheus',
+              queryValue: '',
+              refresh: 1,
+              regex: '/^rhobs.*|telemeter-prod-01-prometheus|app-sre-stage-01-prometheus/',
```
Comment (jacobbaungard) [inline observability/dashboards/rhobs-instance-utilization.libsonnet:60]: Would it be possible to use the variable defined in the config here somehow: https://github.com/rhobs/configuration/blob/5df228ee5846a21b150d41f9cadeedd617feacaf/observability/config.libsonnet#L39 ?

Comment (coleenquadros) [reply observability/dashboards/rhobs-instance-utilization.libsonnet]: I added the variable to utils

Comment (jacobbaungard) [reply observability/dashboards/rhobs-instance-utilization.libsonnet]: Nice, could we just make sure that the line in config.libsonnet uses that variable too? Then if we need to change it, we only have to do it one place :+1: 

https://github.com/rhobs/configuration/blob/82bd76f9e206403ef98c4a20d01043aa319e101b/observability/config.libsonnet#L39

**Review by jacobbaungard — APPROVED** [review]
Cool, looks good to me! (squash commits when merging)

**Review by douglascamata — APPROVED** [review]
LGTM

**Pull request merged**